neuror/sanitize.py

```diff
@@ -41,7 +42,28 @@ def sanitize(input_neuron, output_path):
     fix_non_zero_segments(neuron).write(str(output_path))
 
 
-def sanitize_all(input_folder, output_folder):
+def _sanitize_one(args):
+    '''Function to be called by sanitize_all to catch all exceptions
+    and return path if in error
+
+    Since Pool.imap_unordered only supports one argument, the argument
+    is a tuple: (path, input_folder, output_folder).
+    '''
+    path, input_folder, output_folder = args
```

Comment:
I would use partial to fix the two input/output argument as they are constants

Reply:
Good point